proposals/views/proposal_views.py

```diff
@@ -357,6 +358,19 @@ def get_next_url(self):
     def get_back_url(self):
         """Return to the consent form overview of the last Study"""
         return reverse('proposals:consent', args=(self.object.pk,))
+    
+class ProposalUpdateDataManagement(braces.GroupRequiredMixin, generic.UpdateView):
+    """
+    Allows the secretary to change the Data Management Plan on the Proposal level
+    """
+    model = Proposal
+    template_name = 'studies/study_update_attachments.html'
```

Comment:
Although I appreciate the efficiency of reusing the same template, it's almost more trouble than it's worth because there's context missing now:

![image](https://user-images.githubusercontent.com/7064137/221868028-ada57ce3-2519-4304-8ff9-3c4e2fdb9505.png)